### DISCUSSION TITLE
Same user in multiple orgs with ldap(AD) in grafana 9

### DISCUSSION THREAD

**Discussion by joyartoun** [grafana/grafana#70142] [General]
Hi, 

I am confused with the upgrade from grafana 7 to 9. Before the upgrade, a single user (synced from the AD) could belong to multiple organizations. After the upgrade, when looking up the user in the GUI, it only shows 1 match and a text, see below.

![image](https://github.com/grafana/grafana/assets/29835230/8f5a6798-61ae-4d7d-98ad-ae09bdbf731c)

Is this normal behaviour or have we configured something wrong?

This is a major problem for us.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). This is phrased as a question, and we'd like questions about grafana to go to our[ community forum](https://community.grafana.com/). Feel free to make an [issue](https://github.com/grafana/grafana/issues) if you cannot find a solution and would like to request the feature.
